addons/print/models/print_printer.py

```diff
@@ -249,60 +248,55 @@ def spool_report(self, docids, report_name, data=None, title=None, copies=1):
     @api.model
     def test_page_report(self):
         """Get printer test pages"""
-        Report = self.env['ir.actions.report']
-        return Report.search([
-            ('model', '=', 'print.printer'),
-            ('report_name', '=like', 'print.%'),
-        ])
+        Report = self.env["ir.actions.report"]
+        return Report.search(
+            [
+                ("model", "=", "print.printer"),
+                ("report_name", "=like", "print.%"),
+            ]
+        )
 
-    @api.multi
     def spool_test_page(self):
         """Print test page"""
         for printer in self.printers(raise_if_not_found=True):
-            printer.spool_report(printer.ids, self.test_page_report(),
-                                 title="Test page")
+            printer.spool_report(printer.ids, self.test_page_report(), title="Test page")
         return True
 
-    @api.multi
     def clear_user_default(self):
         """Clear as user default printer"""
         self.env.user.printer_ids -= self
-        return {'type': 'ir.actions.client', 'tag': 'reload'}
+        return {"type": "ir.actions.client", "tag": "reload"}
 
-    @api.multi
     def clear_system_default(self):
         """Clear as system default printer"""
-        self.write({'is_default': False})
-        return {'type': 'ir.actions.client', 'tag': 'reload'}
+        self.write({"is_default": False})
+        return {"type": "ir.actions.client", "tag": "reload"}
 
-    @api.multi
     def set_user_default(self):
         """Set as user default printer (within group, if applicable)"""
         self.ensure_one()
         self.env.user.printer_ids.filtered(
             lambda x: x.group_id == self.group_id and x.report_type == self.report_type
         ).with_env(self.env).clear_user_default()
         self.env.user.printer_ids += self
-        return {'type': 'ir.actions.client', 'tag': 'reload'}
+        return {"type": "ir.actions.client", "tag": "reload"}
 
-    @api.multi
     def set_system_default(self):
         """Set as system default printer (within group, if applicable)"""
         self.ensure_one()
-        self.search([
-            ('is_default', '=', True),
-            ('group_id', '=', self.group_id.id)
-        ]).clear_system_default()
+        self.search(
+            [("is_default", "=", True), ("group_id", "=", self.group_id.id)]
+        ).clear_system_default()
         self.is_default = True
-        return {'type': 'ir.actions.client', 'tag': 'reload'}
+        return {"type": "ir.actions.client", "tag": "reload"}
 
     @api.model
     def clear_ephemeral(self):
         """Clear all ephemeral user default printers"""
-        self.env.user.printer_ids.filtered(
-            lambda x: x.is_ephemeral
-        ).with_env(self.env).clear_user_default()
-        return {'type': 'ir.actions.client', 'tag': 'reload'}
+        self.env.user.printer_ids.filtered(lambda x: x.is_ephemeral).with_env(
```

Comment:
Same here, `lambda p` is better imo as it makes clear straight away that we're dealing with printer records.